package.json

```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint --ignore-path .gitignore .",
-    "test": "npm run lint && npm run mocha",
-    "mocha": "mocha --opts .mocharc",
-    "mocha:watch": "npm run mocha -- --watch"
+    "test": "npm run lint && npm run jest",
+    "jest": "jest --config=.jestrc",
+    "jest:watch": "npm run jest -- --watchAll"
```

Comment:
This can just be `jest --watch`. `watchAll` runs/watches everything, `watch` only watches what's changed (based on git status).